include/pka_config.h

```diff
@@ -53,13 +53,10 @@
 // and ring spacing.
 #define PKA_RING_WORDS_ADDR         PKA_BUFFER_RAM_BASE
 #define PKA_RING_CNTRS_ADDR         COMMAND_COUNT_0_ADDR

```

Comment:
Those should be determined based on the ACPI resources, right?

Reply:
Yes, I'm using them as offsets. The base address defined in ACPI table will be added when using them.